include/rnp/rnp_types.h

```diff
@@ -95,7 +98,11 @@ typedef struct rnp_symmetric_pass_info_t {
 
 /* rnp operation context : contains additional data about the currently ongoing operation */
 typedef struct rnp_ctx_t {
-    rnp_t *        rnp;           /* rnp structure */
+    union {
```

Comment:
No anonymous unions in C99.

(Also probably easier to add at the end of this structure to avoid all the `{0}` -> `{{0}}` changes, but not a particularly big deal)

Reply:
yeah, I know. I was wondering if this should be done in this PR as it will make this PR even bigger. But ... let's do it!

Reply:
Done